test/libsolidity/syntaxTests/types/magic_block_basefee_error.sol

```diff
@@ -2,8 +2,14 @@ contract C {
     function f() public view returns (uint) {
         return block.basefee;
     }
+    function g() public view returns (uint ret) {
+        assembly {
+            ret := basefee()
+        }
+    }
 }
 // ====
-// EVMVersion: <=berlin
+// EVMVersion: =berlin
 // ----
 // TypeError 5921: (74-87): "basefee" is not supported by the VM version.
+// TypeError 5430: (183-190): The "basefee" instruction is only available for London-compatible VMs (you are currently compiling for "berlin").
```

Comment:
@cameel The problem with `<= berlin` is that "you are currently compiling for "berlin"" line would change for each evm version. So it won't pass in CI :)